server/src/game/components/cult.rs

```diff
@@ -0,0 +1,110 @@
+use crate::game::{chat::{ChatGroup, ChatMessageVariant}, phase::PhaseType, player::PlayerReference, role::{apostle::Apostle, disciple::Disciple, zealot::Zealot, RoleState}, role_list::Faction, Game};
+
+
+impl Game {
+    pub fn cult(&self)->&Cult{
+        &self.cult
+    }
+    pub fn set_cult(&mut self, cult: Cult){
+        self.cult = cult;
+    }
+}
+#[derive(Default, Clone)]
+pub struct Cult {
+    pub ordered_cultists: Vec<PlayerReference>,
+    pub sacrifices_needed: Option<u8>
```

Comment:
Calling this `sacrifices_required` works better with line 34, since you can do `if let Some(required)` rather than `Some(s)`. Then on line 35, you can do `CultSacrificesRequired { required }` instead of `CultSacrificesRequired { required: s }`

Reply:
Sure, done.
But also this has nothing to do with this PR

Reply:
Well, it's in the PR, isn't it?

Reply:
True. but i literally just moved a files contents from one spot to another